server-phoenix/mix.exs

```diff
@@ -55,6 +55,7 @@ defmodule Helios.MixProject do
       {:tzdata, "~> 1.1"},
       {:httpoison, "~> 1.7"},
       {:cors_plug, "~> 3.0"}
+      {:sun_times, "~> 0.1.2"}
```

Comment:
This looks like a merge commit with syntax errors. I think you need a comma above this line here. Lets prefer rebase to avoid this.